addons/l10n_bj/data/account_tax_report_data.xml

```diff
@@ -210,12 +210,17 @@
                                 <field name="engine">aggregation</field>
                                 <field name="formula">BJ_VAT_DEDUCTIBLE.balance - BJ_GROSS.balance</field>
                                 <field name="subformula">if_above(EUR(0))</field>
-                        <field name="carryover_target">BJ_REPORTED.balance</field>
+                            </record>
+                            <record id="account_tax_report_line_bj_credit_to_report_carryover" model="account.report.expression">
+                                <field name="label">_carryover_balance</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">BJ_TO_REPORT.balance</field>
+                                <field name="carryover_target">BJ_REPORTED.balance</field>
                             </record>
                         </field>
                     </record>
                 </field>
             </record>
         </field>
     </record>
-</odoo>
\ No newline at end of file
+</odoo>
```

Comment:
Useless diff !!!!!! :skull: :fire: :bomb: :warning: :boom: 

(just kidding, you can keep it)